packages/wonder-blocks-icon-button/components/icon-button.js

```diff
@@ -63,10 +63,11 @@ export type SharedProps = {|
         ...FlexItemStyles,
     }>>,
     */
-|};
 
-type Props = {|
-    ...SharedProps,
+    // NOTE(jeresig): Currently React Docgen (used by Styleguidist) doesn't
+    // support ... inside of an exact object type. Thus we had to move the
+    // following propers into this SharedProps, even though they should be
+    // external. Once that's fixed we can split them back apart.
```

Comment:
Sadness.  I'll try to make some time to investigate this.

Reply:
3.0 beta supports spreads. Maybe we should try that.